core/server/common/src/main/java/alluxio/master/BackupManager.java

```diff
@@ -196,6 +198,7 @@ public void backup(OutputStream os, AtomicLong entryCount) throws IOException {
 
     // finish() instead of close() since close would close os, which is owned by the caller.
     zipStream.finish();
+    es.shutdown();
```

Comment:
do you want to put this in a `final` block so even if the zip stream fails to finish, you still shut down the thread pool?